src/Consolonia.Controls/Markup/OnPlatformExtensions.cs

```diff
@@ -52,9 +54,33 @@ private protected static bool ShouldProvideOptionInternal2(string option)
         {
             return option switch
             {
-                "CONSOLE" => Application.Current.ApplicationLifetime?.GetType().Name == "ConsoloniaLifetime",
+                "CONSOLE" => IsConsole(),
                 _ => AvaloniaMarkup.OnPlatformExtension.ShouldProvideOption(option)
             };
         }
+
+        private static bool IsConsole()
+        {
+#pragma warning disable CA1031 // Do not catch general exception types
+            if (Application.Current?.ApplicationLifetime != null)
+                return Application.Current.ApplicationLifetime.GetType().Name == "ConsoloniaLifetime";
```

Comment:
Can not we just do  " is ConsoloniaLifetime" ? or if exact type is required - "GetType() == typeof(Consolonialifetime)"?